common/shlibs

```diff
@@ -3392,3 +3392,4 @@ libgaminggear.so.0 libgaminggear-0.15.1_1
 libgaminggearfx.so.0 libgaminggear-0.15.1_1
 libgaminggearwidget.so.0 libgaminggear-0.15.1_1
 libopkg.so.1 libopkg-0.4.0_1
+libQMatrixClient.so.0.4 libqmatrixclient-0.4.2.1_1
```

Comment:
are you planning on packaging an application that depends on the library?

Reply:
Yes, I'm planning to separate Quaternion with this package